samples/app/src/androidTest/java/com/sourcepoint/app/v6/TestData.kt

```diff
@@ -206,6 +206,51 @@ object TestData {
 
     val storedConsentV741 = """
         {
+            "sp.key.config.propertyId": 16893,
```

Comment:
Basically we can restore the old test data and for the test that you created we use data with the propertyId